### DISCUSSION TITLE
Lefthand Threading

### DISCUSSION THREAD

**Discussion by aaronknczn** [kachurovskiy/nanoels#99] [Q&A] [answered]
Perhaps a stupid question but is there a way to cut lefthand threads? You just would have to reverse servo direction so such a modification wouldnt be difficult but is that already included in the stock software?

**Comment by kachurovskiy** [comment, marked as answer]
Hi @aaronknczn, to cut left-hand threads set the pitch to a negative value.

Just added a line in https://github.com/kachurovskiy/nanoels/blob/main/h2/README.md#setting-pitch to cover that :)

Cheers!

**Reply by aaronknczn** [reply]
Thanks for the quick reply! Makes sense to do it like this.
Im really excited how my build is gonna work out. Im still waiting on a few parts to show up.
